models/item.js

```diff
@@ -1,19 +1,25 @@
-module.exports = function(sequelize, DataTypes) {
+module.exports = function (sequelize, DataTypes) {
   var Item = sequelize.define("Item", {
-    name: {
+    itemName: {
       type: DataTypes.STRING,
       allowNull: false,
+      validate: {
+        len: [1]
```

Comment:
I believe this form of validation will limit the User's item name to a single letter, right? `min: 1` would at least make it only character but I think having `allowNull: false` should be enough to make sure it isn't blank.

(Unless you were trying to do something else that I'm missing)

Reply:
true, I think I can remove that.